src/utils.jl

```diff
@@ -709,8 +710,8 @@ function Base.StackTraces.StackFrame(frame::Frame)
     end
     Base.StackFrame(
         fname,
-        Symbol(JuliaInterpreter.getfile(frame)),
-        JuliaInterpreter.linenumber(frame),
+        Symbol(getfile(frame)),
+        @something(linenumber(frame), getline(linetable(frame, 1))),
```

Comment:
I couldn't reason that `linenumber(frame)` always returns `Int` here, so added a fallback, which may not be very accurate information but not lead to runtime error at least.